src/Microsoft.AspNetCore.BuildTools.CreateLZMA/Microsoft.AspnetCore.BuildTools.CreateLZMA.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>$description$</description>
+    <copyright>$copyright$</copyright>
+    <packageTypes>
+      <packageType name="MSBuildSdk" />
+    </packageTypes>
+  </metadata>
+  <files>
+    <file src="bin\$config$\netstandard1.5\$id$.dll" target="build\netstandard1.5\$id$.dll" />
```

Comment:
You will also need to copy Microsoft.DotNet.Archive.dll into the package next to the task assembly.